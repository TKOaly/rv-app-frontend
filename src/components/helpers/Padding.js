import React from 'react';

const Padding = (props) => (
    <div
        className="padding"
        style={{
            padding: props.padding,
            display: props.inlineBlock ? 'inline-block' : 'block'
        }}
    >
        {props.children}
    </div>
);

export default Padding;
