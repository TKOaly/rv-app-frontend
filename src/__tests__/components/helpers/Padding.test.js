import Padding from '../../../components/helpers/Padding';
import React from 'react';
import renderer from 'react-test-renderer';

describe.only('<Padding />', () => {
    it('should match snapshot', () => {
        const cmpnt = renderer.create(<Padding />).toJSON();
        expect(cmpnt).toMatchSnapshot();
    });
});
