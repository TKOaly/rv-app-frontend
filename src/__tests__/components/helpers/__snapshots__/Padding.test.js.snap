// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Padding /> should match snapshot 1`] = `
<div
  className="padding"
  style={
    Object {
      "display": "block",
      "padding": undefined,
    }
  }
/>
`;
